### ISSUE TITLE
qs download fails 

### ISSUE THREAD

**Issue by rickydazla** [internalfx/quickshot#58]
```
Nexus6:taylorstitch Rick$ qs theme download
? Select target [live] - 'Master' @ taylor-stitch.myshopify.com
04:05:06 pm - Downloaded 0 files.
Unhandled rejection Error: EEXIST: file already exists, mkdir '/Users/Rick/Sites/myshop
ify/themes/taylorstitch/theme/assets'
```

**Comment by internalfx** [comment]
get `3.0.5` and try it in a fresh folder.

**Comment by rickydazla** [comment]
Re: "fresh folder" ... My general workflow has been to run:
1. `qs watch` on a preview theme
2. `qs download` the live theme
3. commit changes - usually admin-edited `settings_data.json` file and any app-injected snippet updates
4. create new branch and start working on new feature...

Should I now be running `qs sync` instead of `qs download` at step 2?

**Comment by internalfx** [comment]
I would `qs theme download` first

**Comment by rickydazla** [comment]
I can't, see above

**Comment by internalfx** [comment]
dump your quickshot.json

**Comment by internalfx** [comment]
start fresh

**Comment by internalfx** [comment]
You can also jump on gitter...which might be easier for back-and-forth conversation

https://gitter.im/internalfx/quickshot

**Comment by rickydazla** [comment]
Fixed in 3.0.8
